### PR TITLE
Bump to 2.2.6 and enable multiple ruby version testing on Circle

### PR DESCRIPTION
**Pull request by thewalkingtoast**
- Adds multiple ruby version tests for Circle
- Bumps baseline ruby to 2.2.6
- Fix typo in changelog.

### CODE REVIEW

**Comment by KronicDeth** [comment]
Why are we testing on both travis and circle? (I don't remember why we did that) Won't it build faster on travis since the build matrix is actually in parallel while circle does it all in series?

**Comment by thewalkingtoast** [comment]
@KronicDeth We're not using Travis currently. I updated to keep it consistent in case we do switch to Travis. One advantage to using Circle is familiarity and this gem's test suite is pretty small so the extra serial runs of Ruby versions do not significantly impact test time currently.

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 language: ruby
 rvm:
-  - 2.2.1
+  - 2.2.6
```
Comment (shamil614) [inline .travis.yml]: ❤️

File: circle.yml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
     - rabbitmq-server
+dependencies:
+  override:
+    - 'rvm-exec 2.2.6 bundle install'
+    - 'rvm-exec 2.3.3 bundle install'
+    - 'rvm-exec 2.4.0 bundle install'
 test:
   override:
-    - RAILS_ENV=test bundle exec rspec -r rspec_junit_formatter --format RspecJunitFormatter -o $CIRCLE_TEST_REPORTS/rspec/junit.xml
+    - 'RAILS_ENV=test rvm-exec 2.2.6 bundle exec rspec -r rspec_junit_formatter --format RspecJunitFormatter -o $CIRCLE_TEST_REPORTS/rspec/junit.xml'
```
Comment (shamil614) [inline circle.yml]: ❤️  ❤️  ❤️

**Comment by shamil614** [comment]
Excellent improvements. Thank you.

**Pull request merged**